src/GenericController.js

```diff
@@ -0,0 +1,17 @@
+export default class GenericController {
+  constructor() {
+    this.controls = {};
+    const that = this;
+    this.addMethod('init', function init() {
+      that.emitter = this;
+    });
+  }
+
+  addMethod(methodName, method) {
+    this.controls[methodName] = method;
+  }
+
+  getController() {
```

Comment:
Or dodge the issue entirely and move to a utility function instead of a class:

```
function classToGMusicObject(constructor) {
  return Object.assign({
    init: constructor
  }, constructor.prototype);
}

class PlaylistsController {
  // constructor () ...
}
module.exports = classToGMusicObject(PlaylistsController)
```